crew

```diff
@@ -495,11 +495,12 @@ end
 
 def download
   url = @pkg.get_url(@device[:architecture])
+  @external = @pkg.is_external?
```

Comment:
If you want to use a property, you will need to add it to `lib/package.rb` also.  However, if you do this then every package needs this property set to a default value if not set directly.

Reply:
Like `is_fake`, the new `is_external` function in `package.rb` will return `nil` if the variable is not set

Reply:
(which equal to `false`)